src/StreamJsonRpc.Shared/JsonRpc.cs

```diff
@@ -145,7 +145,7 @@ public JsonRpc(Stream sendingStream, Stream receivingStream, object target = nul
         public JsonRpc(DelimitedMessageHandler messageHandler, object target = null)
         {
             Requires.NotNull(messageHandler, nameof(messageHandler));
-
+                        
```

Comment:
Please execute Format Document on this file so that this trailing whitespace (and any other issues) are resolved.